sysutils/colima/Portfile

```diff
@@ -22,7 +26,7 @@ checksums           rmd160  95dd98a7d2120e968c45f057a8af79a186e1c03d \
                     sha256  cfac3e086bbf706b550592fdac787f13b05cc43e361277cf06da59429afb6ea5 \
                     size    608519
 
-depends_run         port:lima
+depends_run         bin:${prefix}/bin/lima:lima port:docker
```

Comment:
This should probably just be `bin:lima:lima` if you want to use lima installed in a different directory in a standard `$PATH`. Otherwise it seems that it's looking for a user-installed `lima` where the port would be installing lima and be no different than `port:lima` in the end.

*Personally*, I’m not as certain about requiring `port:docker` by default, because `colima` has both nerdctl and k8s modes. You are right that the most *common* use of `colima` will be as a replacement for Docker Desktop and installing `docker` alongside it. I’ve been using a Homebrew installed `docker` (mostly because of inertia, but I would have ended up noticing if `docker-buildx` weren't available) with no problem for a while.

Would it make sense to have this and k8s support as variants?

```tcl
variant docker {
  depends_run-append port:docker
}

variant kubernetes {
  depends_run-append port:kubectl-1.29
}
```

I’m less certain of that secondary variant (`kubernetes`) because colima doesn't care about which version of kubectl you are using, but there may be some other restriction on kubectl versions based on configuration (I don't know; I don't use k8s). It might be that there are `kubectl-$VERSION` variants to put in place.

Reply:
I don't think variants are suitable for mere changes in dependencies — without changes in code — as they prevent you from using the nice binary archives.

My goal was to just have a single port that you could install and get a fully functioning Docker runtime. As I think about it, `colima` isn't really that port: That port _should_ be `docker`, just as it is Linux distributions, but doing that is more involved, and beyond the relatively limited scope of this PR.

Reply:
Fair enough. I'm not sure that `docker` should have `colima` as a dependency, because it will work for Docker Desktop as a command-line frontend, too.

Reply:
Seconding the point that `docker` should not be a dependency of `colima`, because of the official Docker Desktop.

Reply:
> Seconding the point that docker should not be a dependency of colima, because of the official Docker Desktop.

Well, that's mostly an alternative in personal or educational contexts, otherwise you have to pay for it. I'll look into refactoring the Docker ports to more closely correspond to what I'd expect: `sudo port install docker` should leave you with something capable of running Docker images.